easybuild/easyconfigs/a/ASAP3/ASAP3-3.12.3-foss-2020b-ASE-3.21.1.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = "PythonPackage"
+
+name = 'ASAP3'
+version = '3.12.3'
+_aseversion = '3.21.1'
+versionsuffix = '-ASE-' + _aseversion
+
+homepage = 'https://wiki.fysik.dtu.dk/asap/'
+description = """ASAP is a calculator for doing large-scale classical molecular
+dynamics within the Campos Atomic Simulation Environment (ASE)."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'pic': True, 'usempi': True, 'openmp': False}
+
+source_urls = [PYPI_LOWER_SOURCE]
```

Comment:
please remove this line, it is now the default for PythonPackages and PythonBundles.